### PR TITLE
Remove LayoutKind.Sequential attributes

### PR DESCRIPTION
**Pull request by AArnott**
These are the default in C# anyway, so it's just noise.

### CODE REVIEW

File: src/User32/User32+MENUINFO.cs [hunk 1]
```diff
@@ -15,9 +15,7 @@ public partial class User32
         /// Contains information about a menu.
         /// </summary>
 #if NETPORTABLE
```
Comment (vbfox) [inline src/User32/User32+MENUINFO.cs]: Empty conditional here, raise the question if we still want `Unicode` or not (I believe not as the windows versions needing it are long gone)

File: src/User32/User32+MENUITEMINFO.cs [hunk 2]
```diff
@@ -15,9 +15,7 @@ public partial class User32
         /// Contains information about a menu item.
         /// </summary>
 #if NETPORTABLE
```
Comment (vbfox) [inline src/User32/User32+MENUITEMINFO.cs]: Same

File: src/User32/User32+MONITORINFOEX.cs [hunk 3]
```diff
@@ -18,9 +18,7 @@ public partial class User32
         /// structure adds a string member to contain a name for the display monitor.
         /// </summary>
 #if NETPORTABLE
-        [StructLayout(LayoutKind.Sequential)]
```
Comment (vbfox) [inline src/User32/User32+MONITORINFOEX.cs:21]: Same

**Review by vbfox — CHANGES_REQUESTED** [review]

File: src/User32/User32+MENUINFO.cs (hunk 1 above)
Comment (AArnott) [reply src/User32/User32+MENUINFO.cs]: Ya, declaring `Unicode` anywhere is deprecated, IMO. Thanks for catching this though, I used regex to remove them all, which didn't bother to tell me what it left behind. :)

**Review by vbfox — APPROVED** [review]

**Pull request merged**